components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FormattingUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.gradle.enterprise.cli;
+
+import java.time.Duration;
+import java.util.Locale;
+
+final class FormattingUtils {
+
+    private FormattingUtils() {
+    }
```

Comment:
A detail, but I usually move the private constructor of classes with no state to the end of the class, since it is not the interesting part of the class.

Reply:
Done in 2cf2c5207417d792b235e778bdf118c179cc8ea7